common/lib/xmodule/xmodule/tests/test_course_metadata_utils.py

```diff
@@ -159,6 +160,10 @@ def mock_strftime_localized(date_time, format_string):
                     advertised_start_unparsable.title()
                 ),
                 TestScenario(
+                    (test_datetime, advertised_start_bad_date, 'DATE_TIME', ugettext, mock_strftime_localized),
```

Comment:
The reason I question is because we're mocking the function.  If we didn't mock the function we would have a better guarantee that we're testing what's run on prod.